ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.js

```diff
@@ -119,8 +125,8 @@ const ConfirmAddSuggestedToken = () => {
   const handleAddTokensClick = useCallback(async () => {
     await Promise.all(
       suggestedAssets.map(async ({ asset, id }) => {
```

Comment:
I'm not all that sure about this `map` with an `async` function inside that awaitsthe `dispatch` but doesn't await `trackEvent`.

I considered returning the three promises with a `flatMap` instead, but I don't want to change existing behaviour so I'm going to assume there's a reason for it (such as not wanting to throw if `trackEvent` fails).